### DISCUSSION TITLE
Programming

### DISCUSSION THREAD

**Discussion by 1HHHHHHHHHH1** [community/community#28080] [Other Feature Feedback, Questions, & Ideas]
Hello, I am new to programming. What is the easiest and best code editor for multilingual programming?

**Comment by zigad** [comment]
Hi! You can start on Youtube. For beginners I recommend Python and then you will see what you like. For Code Editor you can for now use simple text editor or maybe Sublime Text. There are many options there.

If you are also new to Git, you can use my Git Playground repo to try things out without breaking anything: https://github.com/zigad/Playground.

**Comment by n1ckfedorov** [comment]
VSC

**Comment by Josephglz** [comment]
I recommend you use [Visual Studio Code](https://code.visualstudio.com), it has so many plugins that allow you to work with most languages, and its interface is very attractive, adding to this that its performance is good.

**Comment by supervoid0** [comment]
**Visual Studio Code**

Reason to use - 
- VS Code is mostly open-source which alone is a great advantage.
- Has thousands of extensions, themes.
- Supports hundreds of programming languages
- It launches very quickly
- Easy source control
- Syntax highlighting, bracket-matching, auto-indentation, box-selection, snippets, and more.

**Comment by baraaj** [comment]
I recommend visual studio code
